File: POO_Semana1/Pytho_Semana1/Ejercicio01/Person.cpp
#include <iostream>
#include <string>

using namespace std;

class Person {
public:
    string Name;
    int Age;
    string City;

    string describe() {
        return "Name: " + Name + " Age: " + to_string(Age) + " City: " + City;
    }
};

int main() {
    Person person1;
    person1.Name = "Jeremy";
    person1.Age = 20;
    person1.City = "Lima";

    string description = person1.describe();
    cout << description << endl;

    return 0;
}

File: POO_Semana1/Pytho_Semana1/Ejercicio01/Person.py
class Person:
    Name = ""
    Age = 0
    City = ""

    def Describe(self):
        return f"Name: {self.Name} Age: {self.Age} City: {self.City}"


person1 = Person()
person1.Name = "Jeremy"
person1.Age = 20 
person1.City = "Lima"
description = person1.Describe()
print(description)